packages/devtools_app/lib/src/screens/memory/memory_controller.dart

```diff
@@ -256,8 +258,7 @@ class MemoryController extends DisposableController
     });
   }
 
-  /// Memory snapshot taker, that is convinient to mock for testing.
-  late SnapshotTaker snapshotTaker;
+  late DiffPaneController diffPaneController;
```

Comment:
make this late final and add a comment to explain why this is late (I think this is for testing convenience right?)